packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -168,7 +168,16 @@
         margin: 0 !important;
         height: inherit;
     }
-    @include mobile {
+
+    @include new-mobile {
+        .ciq-menu {
+            top: 0;
+            margin: 0.8rem;
+            z-index: 2;
+        }
+    }
+
+    @include new-not-desktop {
```

Comment:
Same, let's not use `new` here, as well as negation. It looks quite confusing also.